docs/user-guide/css-processors.md

```diff
@@ -34,3 +34,29 @@ postcss([
   })
 })
 ```
+
+## Parsing Less
+
+The linter can *parse* Less syntax.
+
+Both the [CLI](/docs/user-guide/cli.md) and the [Node API](docs/user-guide/cli.md) expose a `syntax` option.
+
+- If you're using the CLI, use the `syntax` flag like so:  `stylelint --syntax less ...`
+- If you're using the Node API, pass in the `syntax` option like so: `stylelint.lint({ syntax: "less", ... })`.
+
+If you're using the linter as a [PostCSS Plugin](/docs/user-guide/postcss-plugin.md), you'll need to use [postcss-less](https://github.com/webschik/postcss-less) directly with PostCSS's `syntax` option like so:
+
+```js
+var postcss = require("postcss")
+var less = require("postcss-less")
+
+postcss([
+  require("stylelint"),
+  require("reporter")
+])
+  .process(css, {
+    from: "src/app.css",
+    syntax: less
```

Comment:
Is the user *required* to set those special syntax options `mixinsAsAtRules: true, innerMixinsAsRules: true`? If so, this is the place to document it.